Snake/BusinessLayer/FoodFolder/Food.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace BusinessLayer.FoodFolder
+{
+    public class Food
+    {
+        public int yLocation { get; set; }
+        public int xLocation { get; set; }
+        public int pointsWorth { get; set; }
```

Comment:
Can anything external to the Food classes ever change the pointsWorth? If not consider using a private set here to prevent that from occuring